container/container.go

```diff
@@ -183,6 +184,34 @@ func (container *Container) WriteHostConfig() error {
 	return json.NewEncoder(f).Encode(&container.HostConfig)
 }
 
+// SetupWorkingDirectory sets up the container's working directory as set in container.Config.WorkingDir
+func (container *Container) SetupWorkingDirectory() error {
+	if container.Config.WorkingDir == "" {
+		return nil
+	}
+	container.Config.WorkingDir = filepath.Clean(container.Config.WorkingDir)
```

Comment:
This isn't necessary. `GetResourcePath` deals with that (not to mention that `Clean` doesn't actually clean the path properly in a few cases).